app/models/video.py

```diff
@@ -0,0 +1,20 @@
+from flask import current_app
+from app import db
+from datetime import datetime
+
+class Video(db.Model):
+    video_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime, nullable=True)
+    total_inventory = db.Column(db.Integer, default=0) 
+    available_inventory = db.Column(db.Integer, default=total_inventory) 
```

Comment:
https://docs.sqlalchemy.org/en/13/core/metadata.html#sqlalchemy.schema.Column.params.default

In this line default=total_inventory is not a valid option.